utilities/infra.py

```diff
@@ -44,6 +44,7 @@ def create_ns(
     teardown: bool = True,
     delete_timeout: int = 4 * 60,
     labels: Optional[dict[str, str]] = None,
+    annotations: Optional[dict[str, str]] = None,
```

Comment:
I think we want to move away from option and use the `xx | yy` syntax by importing from future, see https://docs.python.org/3/library/stdtypes.html#types-union

Reply:
done